firebase-firestore/src/main/java/com/google/firebase/firestore/UserDataConverter.java

```diff
@@ -73,14 +73,23 @@ public UserDataConverter(DatabaseId databaseId) {
     this.databaseId = databaseId;
   }
 
-  /** Parse document data from a non-merge set() call. */
+  /**
+   * Parse document data from a non-merge set() call.
+   *
+   * @param input A POJO object representing document data.
```

Comment:
nit: We usually use "POJO object" to refer to custom developer-written classes representing Firestore data, but "input" could also just be a normal Map<String, Object>... which I wouldn't consider a POJO.  So I'd change this to "A Map or POJO object representing document data."

Reply:
Done